hooks/pre-command

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 set -euo pipefail
 
+PLUGIN_NAME=sauce-connect-buildkite-plugin
+
 # Allow to pass TMP_DIR for testing purposes
 if [[ -z "${TMP_DIR:-}" ]]; then
-  TMP_DIR=$(mktemp -d -t sauce-connect-buildkite-plugin.XXXXXXXXXX)
+  TMP_DIR=$(mktemp -d -t "${PLUGIN_NAME}.XXXXXXXXXX")
```

Comment:
This shouldn't use `mktemp` anymore, as that will generate a new random name on each invocation, instead it should be a fixed location, for example `${TMPDIR:-/tmp}/${PLUGIN_NAME}`. Note `TMPDIR` and not `TMP_DIR`: https://en.wikipedia.org/wiki/TMPDIR.

Reply:
3f86ef34997f59fb047fa3a20d309d1134bd87cf